index.html

```diff
@@ -199,14 +199,15 @@ <h2>About me</h2>
         </section>
         <section class="form">
             <div class="formflex">
-                <p>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</p>
+                <h2>I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like to chat please<br>get in touch.</h2>
             <form>
```

Comment:
Your form should submit properly using the [Formspree service](https://formspree.io/html) as required in the description.
- [ ] The action value should be similar to https://formspree.io/f/{form_id}
- [ ] method value equals POST